dogfooding/lib/screens/stats_latency_chart.dart

```diff
@@ -0,0 +1,121 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:fl_chart/fl_chart.dart';
+import 'package:flutter_dogfooding/theme/app_palette.dart';
+
+class StatsLatencyChart extends StatefulWidget {
```

Comment:
Why is this `Stateful`?

Reply:
ah sorry, forgot to switch back to Stateless after changing implementation of it